rainforest-cli_test.go

```diff
@@ -38,6 +45,63 @@ func TestShuffleFlags(t *testing.T) {
 	}
 }
 
+func TestDebugFlag(t *testing.T) {
```

Comment:
I'm not sure how I would create the fakeContext/access related functions in that case?